stock_no_extra_move/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock no extra move",
+    "summary": "Prevent creation of extra moves in picking processing",
+    "version": "8.0.1.0.0",
+    "category": "Logistics",
+    "website": "https://camptocamp.com/",
+    "author": "Camptocamp",
```

Comment:
Can you add "Odoo Community Association (OCA)" here ? This will make travis happy :-)

Reply:
done :)